st2common/st2common/services/workflows.py

```diff
@@ -1123,3 +1123,52 @@ def update_execution_records(wf_ex_db, conductor, update_lv_ac_on_statuses=None,
     if status_changed and wf_lv_ac_db.status in ac_const.LIVEACTION_COMPLETED_STATES:
         LOG.info('[%s] Workflow action execution is completed and invoking post run.', wf_ac_ex_id)
         runners_utils.invoke_post_run(wf_lv_ac_db)
+
+
+@retrying.retry(retry_on_exception=wf_exc.retry_on_exceptions)
+def request_rerun(wf_ex_id, ac_ex_id, st2_ctx, rerun_options):
```

Comment:
Can you keep the method args consistent with other request methods (i.e. request_resume, request_pause, request_cancellation, etc.)?  For example `def request_rerun(ac_ex_db, st2_ctx, options):`